http-kernel-fixtures/print_cookies.php

```diff
@@ -18,9 +18,6 @@
         $cookies['srvr_cookie'] = $srvrCookie;
     }
 
-    foreach ($cookies as $name => $val) {
```

Comment:
I removed this because it was causing this failure:
https://github.com/minkphp/driver-testsuite/actions/runs/5432680290/jobs/9879881814?pr=79

While  `mink_dump` seems to be able to handle different values just fine, I've a feeling this is not backward compatible (e.g. `(int)42` was rendered as `` `42` `` before and now it would be `42`)